agogosml/agogosml/common/eventhub_streaming_client.py

```diff
@@ -111,12 +100,7 @@ def start_receiving(self, on_message_received_callback):
         finally:
             loop.stop()
 
-    def send(self, message):
-        """
-        Send a message to an EventHubStreamingClient.
-
-        :param message: A string input to upload to event hub.
-        """
+    def send(self, message: str):
```

Comment:
Generally, for these streaming clients, I agree that there is a lot of redundant/unnecessary comments. Where can we specify what the param: config should look like -- or at a high level, what each of these streaming clients does? I think we need a description somewhere, yeah?

Reply:
I'd specify the config keys in the constructor comments (as they are being specified now).